src/locale/it-IT.h

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 - Enrico Gueli (@egueli)
+// Locale/language file for Italian.
+// This file will override the default values located in `defaults.h`.
+
+#ifndef LOCALE_IT_IT_H_
+#define LOCALE_IT_IT_H_
+
+#define D_STR_UNKNOWN "SCONOSCIUTO"
+#define D_STR_PROTOCOL "Protocollo"
+#define D_STR_POWER "Accensione"
+#define D_STR_ON "Acceso"
+#define D_STR_OFF "Spento"
+#define D_STR_MODE "Modalità"
+#define D_STR_TOGGLE "Alterna"
+#define D_STR_SLEEP "Stand-by"
```

Comment:
This is the only one that doesn't look right.
"Sleep" here means, the act of sleeping. ie. What you do at night/go to bed. Not suspending a device to low power etc.

Google Translate suggests "Dormire". Correct?

Reply:
Are you sure about that? I saw some usages of `kSleepStr` and, by looking at the surrounding lines, they all seem related to some device mode, e.g. https://github.com/crankyoldgit/IRremoteESP8266/blob/bbf3738191982323da69f4433e89d84238b23caf/src/ir_Daikin.cpp#L2715, https://github.com/crankyoldgit/IRremoteESP8266/blob/f5a517464c5d870f811a7adc8bf6e0b2435b803a/src/ir_Neoclima.cpp#L426 or https://github.com/crankyoldgit/IRremoteESP8266/blob/bbf3738191982323da69f4433e89d84238b23caf/src/ir_Goodweather.cpp#L353

Reply:
Yes. It is often a special device mode. But it is a mode for when you are in bed sleeping.
e.g.
The mode on a number of devices will after a period, start to slowly increase the temperature (if in cooling mode) so it becomes more energy efficient etc. It does this till it reaches a certain level and either stays that way, or in some cases, turns off.

Hence, while it could be seen as a "a mode that will turn off" i.e. if or how long before it run for until it turns off. They often do more than just that. They are commonly used/exist for a mode while _you_ are sleeping, not so much the device.

Reply:
OK, I translated with "Sonno".

Reply:
Thanks